mysite/code/tasks/DeleteRedundantAddonsTask.php

```diff
@@ -1,37 +1,53 @@
 <?php
+
+use Elastica\Exception\NotFoundException;
+
 /**
  * Deletes packages removed from Packagist.
+ *
+ * @package mysite
  */
-class DeleteRedundantAddonsTask extends BuildTask {
-
-	protected $title = 'Delete Redundant Add-ons ';
-
-	protected $description = 'Deletes packages removed from Packagist';
-
-	public function run($request) {
-		$dateOneWeekAgo  = date('Y-m-d', strtotime('-1 week'));
-
-		$addons = Addon::get()->filter('LastUpdated:LessThan' , $dateOneWeekAgo);
-
-		foreach ($addons as $addon) {
-			try {
-				$addon->Keywords()->removeAll();
-				$addon->Screenshots()->removeAll();
-				$addon->CompatibleVersions()->removeAll();
-
-				foreach ($addon->Versions() as $version) {
-					$version->Authors()->removeAll();
-					$version->Keywords()->removeAll();
-					$version->CompatibleVersions()->removeAll();
-					$version->delete();
-				}
-
-				$addon->delete();
-			} catch(Elastica\Exception\NotFoundException $e) {
-				// no-op
-			}
-			
-		}
-	}
-
+class DeleteRedundantAddonsTask extends BuildTask
+{
+    /**
+     * {@inheritDoc}
+     * @var string
+     */
+    protected $title = 'Delete Redundant Add-ons';
+
+    /**
+     * {@inheritDoc}
+     * @var string
+     */
+    protected $description = 'Deletes packages removed from Packagist';
+
+    /**
+     * {@inheritDoc}
+     * @param SS_HTTPRequest $request
+     */
+    public function run($request)
+    {
+        $dateOneWeekAgo  = date('Y-m-d', strtotime('-1 week'));
+
+        $addons = Addon::get()->filter('LastUpdated:LessThan', $dateOneWeekAgo);
+
+        foreach ($addons as $addon) { /** @var Addon $addon */
+            try {
+                $addon->Keywords()->removeAll();
```

Comment:
I hadn't actually looked much at these parts, it's mostly showing these lines up because of indentation changes